build-test.sh

```diff
@@ -270,24 +266,26 @@ build_Tests()
         fi
     fi
 
-    echo "Starting the Managed Tests Build..."
-
-    build_MSBuild_projects "Tests_Managed" "$__ProjectDir/tests/build.proj" "Managed tests build (build tests)" "$__up"
-
-    if [ $? -ne 0 ]; then
-        echo "${__MsgPrefix}Error: build failed. Refer to the build log files for details (above)"
-        exit 1
-    else
-        echo "Checking the Managed Tests Build..."
+    if [ $__SkipManaged != 1 ]; then
+        echo "Starting the Managed Tests Build..."
 
-        build_MSBuild_projects "Check_Test_Build" "${__ProjectDir}/tests/runtest.proj" "Check Test Build" "-ExtraParameters:/t:CheckTestBuild"
+        build_MSBuild_projects "Tests_Managed" "$__ProjectDir/tests/build.proj" "Managed tests build (build tests)" "$__up"
```

Comment:
Please keep braces for variable names (i.e. `${__ProjectDir}`) in bash.

Reply:
This is a whitespace diff, only has been indented.

Reply:
Left side:

> `build_MSBuild_projects "Check_Test_Build" "${__ProjectDir}/tests/runtest.proj" "Check Test Build" "-ExtraParameters:/t:CheckTestBuild"`

Right side:

> `build_MSBuild_projects "Tests_Managed" "$__ProjectDir/tests/build.proj" "Managed tests build (build tests)" "$__up"`

The above is what I am seeing on github. If your changes are not removing the braces around `__ProjectDir`, then that is fine, but in general we should not be removing braces from environment variable access that are embedded in strings.

Reply:
Correct, this variable already was missing the braces. None have been removed. We should most likely add some; however, I will consider that out of scope for this change.